libfreerdp/codec/progressive.c

```diff
@@ -1771,14 +1771,13 @@ static INLINE SSIZE_T progressive_process_tiles(PROGRESSIVE_CONTEXT* progressive
 	for (index = 0; index < region->numTiles; index++)
 	{
 		RFX_PROGRESSIVE_TILE* tile = region->tiles[index];
-		PROGRESSIVE_TILE_PROCESS_WORK_PARAM* param = &params[index];
```

Comment:
these changes do not make any sense.
1. the `params` are allocated directly before the loop
2. the size is `region->numTiles` so out of bound writes are impossible.

Reply:
@akallabeth Sorry, I specified wrong destination branch initially.
I changed it to **stable-2.0**.

Could you review once again?